scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
They are optional, however we don't want to simply remove every optional dependency, but platform specific optional dependencies, and optional dependencies of platform specific optional dependencies.